src/main/java/appeng/client/gui/implementations/GuiCraftConfirm.java

```diff
@@ -60,7 +64,20 @@
 
 	private final ContainerCraftConfirm ccc;
 
-	private final int rows = 5;
+	private int rows = 5;
+	private int maxRows = Integer.MAX_VALUE;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the "maxRows" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1450)